metapool/tests/test_plate.py

```diff
@@ -563,6 +565,109 @@ def test_sample_plate_has_no_spaces(self):
         self.assertEqual(context, expected)
 
 
+class PlateReplicationTests(TestCase):
+    def setUp(self):
+        self.input_df = pd.read_csv('metapool/tests/data/input_plate.tsv',
+                                    sep='\t', dtype=str)
+
+    def test_overwrite_source_quad(self):
+        # replicate a valid source to empty sources 2 and 4 plus overwriting
+        # source 3.
+
+        pr = PlateReplication('Library Well')
+
+        obs = pr.make_replicates(self.input_df, {1: [2, 3, 4]}, overwrite=True)
+
+        exp = pd.read_csv('metapool/tests/data/file1.tsv',
+                          sep='\t', dtype=str)
+
+        obs = obs.set_index('Sample').sort_index()
+        exp = exp.set_index('Sample').sort_index()
+
```

Comment:
Do we need to set_index/sort even if we are using `assert_frame_equal`? I would think no but not sure. This also applies for the rest of the tests.

Reply:
Confirmed we do need, unless we use the check_like flag. I replaced the sort()s with the flag on all of the tests, ty!